### ISSUE TITLE
step_count in the generator loop

### ISSUE THREAD

**Issue by lihuiknight** [POSTECH-CVLab/PyTorch-StudioGAN#65]
Dear author,

I notice that the step_count += 1 is in the generator training loop (see [line](https://github.com/POSTECH-CVLab/PyTorch-StudioGAN/blob/66309a7c1c0f649e9bf089303d08e04aefff1431/src/worker.py#L442) ). 
Does this step count is a standard usage in the research field of GAN.

Specifically, if comparing different methods over the steps, the x-axis is the the number of training steps of generator?

**Comment by MINGUKKANG** [comment]
Hi,

It depends on the paper. 

In the case of ["Consistency regularization for Generative Adversarial Networks"](https://arxiv.org/pdf/1910.12027.pdf), they have written that "We stop training after 200k generator update steps for CIFAR10 and 100k steps for CelebA".

Thank you.

Sincerely,

**Comment by lihuiknight** [comment]
ok，i see, thanks.
